electron-builder.json

```diff
@@ -39,7 +39,7 @@
     "icon": "./dist/static/AppIcon.ico"
   },
   "linux": {
-    "artifactName": "${productName}.${ext}",
+    "artifactName": "${productName}-${version}.${ext}",
```

Comment:
The artifacts for macOS and Windows had this `${version}` portion in their filenames so I added it here as well for consistency. It's been handy with the nightly builds since I can see the commit in the filenames accumulating in my `Downloads/` directory.